orc8r/cloud/go/services/bootstrapper/servicers/registration/registration.go

```diff
@@ -76,37 +84,6 @@ func RegisterDevice(deviceInfo *protos.GatewayDeviceInfo, hwid *protos.AccessGat
 	return err
 }
 
-func GetControlProxy(networkID string) (string, error) {
```

Comment:
We could leave the function in the same position and just change the content.

Reply:
I moved it back to where it was.